build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -22,6 +22,9 @@
     "//llvm-external-projects/iree-dialects:IREELinalgExtPasses": [
         "IREELinalgExtPasses"
     ],
+    "//third_party/torch-mlir-dialects:TorchMLIRTMTensorDialect": [
```

Comment:
This needs to be set up as a Bazel external repository. Is the BUILD file there copied from upstream? If not, we should put it in the build tools third party overlay directory. Here's an example:

https://github.com/google/iree/blob/47e45d3c433a4b20eb9ef28c2340e47ae7108ca0/build_tools/bazel/workspace.bzl#L52-L57

```suggestion
    "@torch-mlir-dialects//:TorchMLIRTMTensorDialect": [
```